version-published/snap_info_utility.py

```diff
@@ -65,6 +65,7 @@ def get_version_and_offset(version_str: str):
     # Try to parse the version and dev number
     try:
         Version(base_version)
+        dev_number = int(dev_number)
```

Comment:
There's some repetition here, the function itself returns `int(dev_number)`.

Reply:
Yes, I moved the `int()` inside the try so it fails beforehand if the dev number is not an integer. I will remove the second one